src/net_processing.cpp

```diff
@@ -337,7 +337,7 @@ class PeerManagerImpl final : public PeerManager
     void MaybeSendPing(CNode& node_to, Peer& peer);
 
     /** Send `addr` messages on a regular schedule*/
-    void MaybeSendAddr(CNode* pto);
+    void MaybeSendAddr(CNode& pto);
```

Comment:
Maybe change this to `node` or `node_to` as per `MaybeSendPing`. It's no longer a `p` either way.

Reply:
Done